nf-test.config

```diff
@@ -11,9 +11,6 @@ config {
     // location of an optional nextflow.config file specific for executing tests
     configFile "nextflow.config"
 
-    // run all test with the defined docker profile from the main nextflow.config
-    profile "docker"
-
```

Comment:
I removed it because the CI tests command already adds the +docker, so you would get a command like --profile docker,test,docker and when running the tests with singularity you would get --profile docker,test,singularity, which doesn't look good to me and even though it works, I fear may cause issues at some point.

I agree that there are way too many ways of running nf-tests, so I wanted to keep things similar to what we did in smrnaseq. This way there can't be any "secret" profiles when running the nf-test command, and one can overwrite them all directly from the command line. I removed the +test_profile from the nf-tests, leaving only the test profile also for consistency.